src/Microsoft.VisualBasic/src/MatchingRefApiCompatBaseline.netstandard.txt

```diff
@@ -1,16 +1,7 @@
 Compat issues with assembly Microsoft.VisualBasic:
-TypesMustExist : Type 'Microsoft.VisualBasic.ControlChars' does not exist in the implementation but it does exist in the contract.
-CannotSealType : Type 'Microsoft.VisualBasic.HideModuleNameAttribute' is sealed in the implementation but not sealed in the contract.
-TypesMustExist : Type 'Microsoft.VisualBasic.Interaction' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'Microsoft.VisualBasic.Strings.Left(System.String, System.Int32)' does not exist in the implementation but it does exist in the contract.
+
+# Loaded via reflection, so it should not be added to the ref assembly:
```

Comment:
Just for clarity I would change the comment to "Loaded via reflection so needs to be public in the implementation but we don't want to expose them publicly in the ref assembly"